services/nexus/nexus.tester.js

```diff
@@ -7,34 +7,34 @@ const t = (module.exports = require('../tester').createServiceTester())
 
 t.create('search release version valid artifact')
   .timeout(15000)
-  .get('/r/https/oss.sonatype.org/com.google.guava/guava.json')
+  .get('/r/com.google/bitcoinj.json?server=https://oss.sonatype.org')
```

Comment:
Our Nexus tests are notoriously bad about timeouts. I found using this target was noticeably faster though (~1 second vs. 10-12 seconds)